src/server/middleware/saveStoryWithBinaries.js

```diff
@@ -41,7 +42,10 @@ fluid.defaults("sjrk.storyTelling.server.middleware.saveStoryWithBinaries", {
     }
 });
 
-// Renames any uploaded files to a pattern of uuid + extension
+/* Renames any uploaded files to a pattern of uuid + extension
```

Comment:
GPII and infusion projects are using [JSDoc|https://jsdoc.app/] style of comments. These comments can be scanned by JSDoc tools to generate static documentation. You probably should consider to use the same pattern.

Reply:
I thought about doing that as well, but figured that might be more appropriately done in its own Jira. It's an interesting question though, and I did make an effort to make the comments on all functions quasi-JSDoc like and consistent with each other so that the work to convert them wouldn't be too difficult.

Do you think we should take this on as part of this Jira or leave it for later?

Reply:
If we decide to use JSDoc style, it might be less effort from now on to apply it to any new or changed comments in each pull request. It would be a lots of effort to add comments for all functions, but for comments you are adding/modifying in this pull request, it would be a good start to apply JSDoc in it.

Reply:
Oh, I did the work to add comments for all functions as part of this PR :) so it's more just a matter of converting the style from my custom one to JSDoc style